paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/AppearanceBottomSheetDialogFragment.kt

```diff
@@ -800,3 +1039,68 @@ private fun getFontFromResource(fontResId: Int?): FontFamily {
         FontFamily.Default
     }
 }
+
+@OptIn(ExperimentalEmbeddedPaymentElementApi::class)
+private fun resetEmbeddedAppearance(
+    viewModel: PaymentSheetPlaygroundViewModel,
+    currentAppearance: PaymentSheet.Appearance,
+    updateAppearance: (PaymentSheet.Appearance) -> Unit
+) {
+    viewModel.updateEmbeddedAppearance(
```

Comment:
Can't just iterate over a list of these and call `updateEmbeddedAppearance` as I can't cast to the captured type